src/Tecan/BasicCommands/AspirateAndDispenseParameters.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Mll\LiquidHandlingRobotics\Tecan\BasicCommands;
+
+use Mll\LiquidHandlingRobotics\Tecan\Rack\Rack;
+
+final class AspirateAndDispenseParameters
+{
+    private Rack $rack;
+
+    private int $startPosition;
+
+    private int $endPosition;
+
+    public function __construct(Rack $rack, int $startPosition, int $endPosition)
+    {
+        $this->rack = $rack;
+        $this->startPosition = $startPosition;
+        $this->endPosition = $endPosition;
+    }
+
+    public function formatToString(): string
+    {
+        return $this->rack->name() . ';'
```

Comment:
`implode(';'` would express the intent more clearly in my opinion.

Reply:
I don't think I'm understanding you correctly. Can you explain in more detail what you mean?

Reply:
I suggest to use `implode` instead of string concatenation, since this is semantically what happens: multiple segments are joined with a single separator.